spiffs_file_preprocessor/txt_processor.c

```diff
@@ -7,6 +7,20 @@
 #include "heatshrink_encoder.h"
 #include "fileUtils.h"
 
+long remove_chars(char* str, long len, char c) {
```

Comment:
It is unclear whether `len` is the `strlen` (excluding terminator) or `sizeof` (including null terminator). Looking at how the function is used, I think it's the `strlen` size, but the loop also checks the null terminator. Versatile? You should make a doxygen comment with what args should be and what behavior is.

Reply:
Done